p2p/discover/table.go

```diff
@@ -58,10 +61,17 @@ type bucket struct {
 	entries    []*Node
 }
 
-func newTable(t transport, ourID NodeID, ourAddr *net.UDPAddr) *Table {
+func newTable(t transport, ourID NodeID, ourAddr *net.UDPAddr, nodeDBPath string) *Table {
+	// If no seed cache was given, use an in-memory one
+	db, err := newNodeDB(nodeDBPath)
+	if err != nil {
+		glog.V(logger.Warn).Infoln("Failed to open node database:", err)
+		db, _ = newNodeDB("")
+	}
+	// Create the bootstrap table
```

Comment:
This comment is incorrect. This is an ordinary table, not a 'bootstrap' table ;).